index.html

```diff
@@ -2766,134 +2728,114 @@ <h3>Consumption</h3>
-    "type": "ed25519Signature2018"
-  },
-  "verificationMethod": [
-    {
-      "curve": "ed25519",
-      "expires": "2019-12-01T03:00:00Z",
-      "id": "bafyreicubtx5wqo3nosc4cazrkctfhwd6rewezgpwoe4swirls4ebdhs2i;example:key=idbafyreicubtx5wqo3nosc4cazrkctfhwd6rewezgpwoe4swirls4ebdhs2i",
-      "publicKeyBase64": "qmz7tpLNKKKdl7cD7PbejDiBVp7ONpmZbfmc7cEK9mg=",
-      "type": "EdDsaPublicKey"
-    }
-  ],
-  "updated": "2020-05-01T03:00:00Z"
-}
+<pre class="example nohighlight" title="DID Document as DagCBOR with tag 42 used in Proof section">
+  a5                                      # map(5)
+  62                                   # text(2)
+     6964                              # "id"
+  78 40                                # text(64)
+     6469643a6578616d706c653a313244334b6f6f574d4864727a6377706a6264725a7335474771455241766367715833623564707550745061396f743639796577 # "did:example:12D3KooWMHdrzcwpjbdrZs5GGqERAvcgqX3b5dpuPtPa9ot69yew"
+  65                                   # text(5)
+     70726f6f66                        # "proof"
+  d8 2a                                # tag(42)
+     58 26                             # bytes(38)
+        000190011b20a7f3a13d0dafa9b8b640fee52173c27b9a66ff27f6669b3be73fc0ee6849e2ce # "\x00\x01\x90\x01\e \xA7\xF3\xA1=\r\xAF\xA9\xB8\xB6@\xFE\xE5!s\xC2{\x9Af\xFF'\xF6f\x9B;\xE7?\xC0\xEEhI\xE2\xCE"
+  68                                   # text(8)
+     40636f6e74657874                  # "@context"
+  78 1c                                # text(28)
+     68747470733a2f2f7777772e77332e6f72672f6e732f6469642f7631 # "https://www.w3.org/ns/did/v1"
+  6e                                   # text(14)
+     61757468656e7469636174696f6e      # "authentication"
+  81                                   # array(1)
+     78 83                             # text(131)
+        6469643a6578616d706c653a313244334b6f6f574d4864727a6377706a6264725a7335474771455241766367715833623564707550745061396f7436397965773b6b65792d69643d6261667972656963756274783577716f336e6f73633463617a726b63746668776436726577657a6770776f6534737769726c733465626468733269 # "did:example:12D3KooWMHdrzcwpjbdrZs5GGqERAvcgqX3b5dpuPtPa9ot69yew;key-id=bafyreicubtx5wqo3nosc4cazrkctfhwd6rewezgpwoe4swirls4ebdhs2i"
+  72                                   # text(18)
+     766572696669636174696f6e4d6574686f64 # "verificationMethod"
+  81                                   # array(1)
+     78 40                             # text(64)
+        6469643a6578616d706c653a313244334b6f6f574d4864727a6377706a6264725a7335474771455241766367715833623564707550745061396f743639796577 # "did:example:12D3KooWMHdrzcwpjbdrZs5GGqERAvcgqX3b5dpuPtPa9ot69yew"
           </pre>
         </section>
+      </section>
+    </section>
+    <section>
+      <h3>Converting Data between Core Representations</h3>
+      <p></p>
 
-        <section>
-          <h4>COSE signatures</h4>
+    <section>
+      <h3>CBOR to JSON</h3>
```

Comment:
Yep, highly problematic because this is going from CBOR right through to JSON -- not via Abstract Data Model. if we go this route, we'll have to define NxN algorithms for each representation to go to each other representation.

Reply:
I expect every developer who reads the spec to essentially do this... the ADM is the kind of complexity people avoid reading, and just guess at... in this case, the guess accurate, you can in fact convert CBOR to JSON or vice versa....without reading about INFRA or the opinions of this WG on data models... we should avoid making the "guessing" incompatible with normative statements, but I think it might also be wise to specifically call out sections like this with a disclaimer that "you have to go through the ADM to convert CBOR to JSON".

Reply:
There is also a section in the CBOR spec that speaks to this -- can't we just refer to that section? 

https://tools.ietf.org/html/rfc7049#section-4

Reply:
Please refer to https://www.ietf.org/archive/id/draft-ietf-cbor-7049bis-16.html which is already in RFC-EDITOR stage and probably will have an RFC number before you are done.

Reply:
I second @msporny 's concerns. 
Representation descriptions should define production and consumption rules to and from the ADM, and should not include translation rules between specific representations.

Reply:
I would like to argue for CDDL to facilitate the data representation aspects of the Abstract Data Model and thus JSON <--> CDDL <--> CBOR.  Infra just isn't doing it and I think CDDL will.